package.json

```diff
@@ -16,11 +16,12 @@
     "consolidate": "0.x",
     "express": "4.13.3",
     "express-nunjucks": "^0.9.3",
+    "express-session": "^1.13.0",
     "express-writer": "0.0.4",
     "govuk-elements-sass": "alphagov/govuk_elements#v1.1.1",
     "govuk_frontend_toolkit": "^4.6.0",
-    "govuk_template_mustache": "^0.16.4",
     "govuk_template_jinja": "https://github.com/alphagov/govuk_template/releases/download/v0.16.4/jinja_govuk_template-0.16.4.tgz",
+    "govuk_template_mustache": "^0.16.4",
```

Comment:
I think it's ok here because there is a valid change to package.json (adding session), whereas I don't think yours had a change?